src/cheri_insts.sail

```diff
@@ -716,15 +716,39 @@ function clause execute (CSEQX(rd, cs1, cs2)) = {
   RETIRE_SUCCESS
 }
 
-  /*
-   * Common suffix of CSeal and CCSeal
-   */
-val CSeal_core : (regidx, regidx, Capability, regidx, Capability) -> Retired effect { rreg, wreg, escape }
-function CSeal_core (cd, cs1, cs1_val, cs2, cs2_val) = {
+union clause ast = CSeal : (regidx, regidx, regidx)
+/*!
+ * Capability register *cd* is replaced with capability register *cs1*, and is
+ * sealed with **otype** equal to the **address** field of capability register
+ * *cs2*.
+ *
+ * ## Exceptions
+ *
+ * An exception is raised if:
+ *   - *cs1*.**tag** is not set.
+ *   - *cs2*.**tag** is not set.
+ *   - *cs1*.**sealed** is set.
+ *   - *cs2*.**sealed** is set.
+ *   - *cs2*.**permit\_seal** is not set.
```

Comment:
```suggestion
 *   - *cs2*.**permit_seal** is not set.
```
Do we have to escape the underscore? There aren't any more underscores in this line, so you'd hope the markdown parser doesn't do weird stuff.
It seems like sail uses a markdown parser that hasn't had a release since 2017 (https://opam.ocaml.org/packages/omd/#), but it seems like the recent git version claim to be commonmark 0.29 compliant. The spec says `Emphasis with _ is not allowed inside words`, so if this is a bug maybe we can upgrade the sail markdown parser to a git snapshot?

Reply:
I also wonder if we should be phrasing this more like `*cs* does not grant the Permit_Seal permission` in case we decide to merge redundant permission bits?